next.config.js

```diff
@@ -28,17 +39,10 @@ module.exports = withBundleAnalyzer({
 
   // Redirects configuration
   async redirects() {
-    return [
-      {
-        source: '/kommuner',
-        destination: '/',
-        permanent: true,
-      },
-      {
-        source: '/kommuner/kommun/ume%C3%A5/parisavtalet',
-        destination: '/',
-        permanent: true,
-      },
-    ]
+    return paths.map((path) => ({
+      source: `/${path}/:slug*`,
+      destination: '/',
+      permanent: true,
+    }))
```

Comment:
This functions as intended by the code, always redirecting to `/` if you try to open an old route.

We discussed sending people to the start page so they could see the new company data. They can always navigate to the other datasets if that's what they are looking for.

This is ready to merge.